src/index.html

```diff
@@ -7,12 +7,55 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
```

Comment:
Do we need to import all weights if we use only 3?
```suggestion
           href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
```